content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -1,75 +1,77 @@
 ---
-title: Poseidon-Firmament - An alternate scheduler
+title: Poseidon/Firmament scheduler
```

Comment:
nit: Poseidon/Firmament -> Poseidon-Firmament. 
As per input from @deepak-vij, unless there is a style guide that enforces the changes being made.

Reply:
Fixed